tests/conftest.py

```diff
@@ -325,3 +331,54 @@ def render_span(span):
         return "\n".join(render_span(root_span))
 
     return inner
+
+
+@pytest.fixture(name="Any")
+def make_any():
+    class Any(object):
+        """
+        An object which matches any instance of the class passed to __init__.
+
+        Useful for assert_called_with statements.
+        """
```

Comment:
Putting the docstring on fixture function will make it show up with `pytest --fixtures` which is friendlier

Reply:
Good to know - thanks!